src/website/app/demos/Dropdown/examples/renderItem.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+
+import Button from '../../../../../library/Button';
+import Dropdown from '../../../../../library/Dropdown';
+import renderItem from '../../Menu/components/renderItem';
+import renderPropDescription from '../../shared/renderPropDescription';
+
+export default {
+  id: 'render-item',
+  title: 'Custom Rendering - renderItem',
+  description: `Use the \`renderItem\`
+[render prop](https://reactjs.org/docs/render-props.html) to provide custom
+rendering control of all [MenuItems](/components/menu-item) in the Menu.
+
+The implementation of \`renderItem\` used in the following example can be seen
+in full in the [Menu](/components/menu/#render-item) example.
```

Comment:
I think it would be helpful to call this info out, both here and in the Select example.

Reply:
It was in there, but I was asked to remove it.  I agree with you though - I'll add it back in.